model_analyzer/analyzer/perf_analyzer/perf_throughput.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+from model_analyzer.record.record import Record
+from model_analyzer.record.perf_record import PerfRecord
+from model_analyzer.model_analyzer_exceptions import TritonModelAnalyzerException
+
+
+class PerfThroughput(Record, PerfRecord):
+    """
+    A record for perf_analyzer
+    metric 'Throughput'
+    """
+    def __init__(self, throughput, perf_config):
+        self._throughput = throughput
+        self._config = perf_config
+
+    def header(self):
+        """
+        Returns
+        -------
+        str
+            The full name of the
+            metric.
+        """
+        return "Throughput (infer/sec)"
+
+    def value(self):
+        """
+        This method returns
+        the value of recorded
+        metric
+
+        Returns
+        -------
+        (any)
+            value of the metric
+        """
+        return self._throughput
+
+    def config(self):
```

Comment:
Do we want config to be a part of the record?

Reply:
I think its useful to have the config for all the perf metrics, that way we can get the batch and concurrency that they were run with whenever needed.